node-graphql/server.schema.js

```diff
@@ -241,7 +241,7 @@ const typeDefs = gql`
     updateExperimenterUser(username: String, isExperimenter: Boolean): JSON,
     uploadSurveyResults(surveyId: String, results: JSON): JSON,
     uploadScenarioResults(results: [JSON]): JSON,
-    addNewParticipantToLog(participantData: JSON): JSON,
+    addNewParticipantToLog(participantData: JSON, lowPid: Int, highPid: Int): JSON,
```

Comment:
Are these needed? I was trying to track them down in the mutation and didn't find them.

Reply:
I know past me knew they were needed, and I think present me agrees, but it looks like the addition of them got forgotten. The low one, at least, should replace 202411301 in line 581. Maybe we don't need high...thoughts?

Reply:
discussed on teams